### PR TITLE
Fix a couple of Cygwin breakages

### PR DESCRIPTION
**Pull request by dra27**
Still testing locally...

### CODE REVIEW

**Comment by yallop** [comment]
Thanks for fixing this, @dra27.  Is it possible to change the CI configuration (e.g. with an additional AppVeyor build) so that problems like this are caught earlier?

**Comment by dra27** [comment]
Normally Inria's trunk CI would have caught it! We could add another AppVeyor build, though it's already taking 30-40 mins! Let's hang fire until more CI comes online later in the year...

File: configure [hunk 1]
```diff
@@ -1840,6 +1840,10 @@ if $shared_libraries_supported && ./hasgot -DPACKAGE=ocaml -i bfd.h ; then
     LIBBFD_LINK="-lbfd -ldl -liberty -lz"
     inf "BFD links with $LIBBFD_LINK"
     echo "#define HAS_LIBBFD" >> s.h
+  elif sh ./hasgot -DPACKAGE=ocaml -lbfd -liberty -lintl -lz bfd_openr; then
```
Comment (yallop) [inline configure]: This doesn't seem to be related to #1274.

Comment (dra27) [reply configure]: No, this part's not - I just pushed it in while sorting out Cygwin!

**Comment by dra27** [comment]
The BFD change has opened a can of worms (ocamlobjinfo doesn't work on the Cygwin port) which will merit an entirely separate piece of work. This is ready to go, if another dev could be so kind as to approve it. The gitattributes tweaks are obviously not related to #1274, but to my Windows git checkout...

**Review by gasche — APPROVED** [review]
